include/miopen/miopen.h

```diff
@@ -5310,19 +5310,20 @@ typedef enum
     miopenTensorConvolutionX      = 1,
     miopenTensorConvolutionW      = 2,
     miopenTensorConvolutionY      = 3,
+    miopenTensorSoftmaxX  = 4,
```

Comment:
[Informative] I suspect that clang-format will fail. See https://github.com/ROCm/MIOpen/wiki/How-to-format-code